package.json

```diff
@@ -4,7 +4,7 @@
 	"description": "This is a Workshop about Api Testing in JavaScript",
 	"main": "index.js",
 	"scripts": {
-		"test": "mocha -t 5000"
+	"test": "mocha -t 5000 --reporter mochawesome --reporter-options reportDir=report,reportFilename=ApiTesting"
```

Comment:
Please don't remove the identation, just add one tab space. Do the same for other nested lines as: type, chai, http-status-codes and so on